src/components/features/hero-banner/HeroBanner.tsx

```diff
@@ -85,6 +87,12 @@ export const HeroBanner = ({
         maxHeight={{ base: '50vh', lg: '80vh' }}>
         {heroBannerImage?.url && (
           <CtfImage
+            //TODO: tagging isn't working here
+            // livePreviewProps={ContentfulLivePreview.getProps({
+            //   entryId,
+            //   fieldId: 'heroBannerImage',
+            //   locale: router.locale,
+            // })}
```

Comment:
If anyone knows how to make the tagging work on a full-width image (mostly asking Tolkien people here) then yes, otherwise I'll just delete it before merging

Reply:
Could you please take care of removing that comment in a follow-up task on the template? Thanks!

Reply:
This is outdated, I found a way to do it and this comment was removed a few commits ago already. Also on the private PR

Reply:
My bad, I had to give the page a refresh. Awesome that you found the way!